FrontEnd/src/app/component/navbar/navbar.component.html

```diff
@@ -1,12 +1,9 @@
-<a routerLink = "">home</a>
-<a routerLink = "login">login</a>
-<p>navbar works!</p>
 <mat-toolbar>
  <span>My Weather</span>
  <span class="example-spacer"></span>
  <div class="button-row">
-  <button mat-raised-button color="primary">Today</button>
-  <button mat-raised-button color="primary">Hourly</button>
+  <button routerLink = "" mat-raised-button color="primary">home</button>
```

Comment:
I would prefer to use the icon for login and home and keep the buttons for the weather. the site would look much better in my opinion.